### ISSUE TITLE
Toggle for sound on connect

### ISSUE THREAD

**Issue by longus-catus** [connectbot/connectbot#691]
I didn't see it in the options and didn't see anyone mention it so I guess I'm in the minority.  

I find the notification sound when connecting to a host really annoying...especially when theres trouble with a host I'm connecting to.  That's when and why I break out connectbot.  Is there a way other than changing system volume settings to mute this?  

Uh, I guess since I'm at it, knowing what precisely it signifies would be nice...connect succeed, first byte received, handshake complete, etc...

It would make this one person happy to be able to disable it and continue to use the main line source.

**Comment by ratermir** [comment]
I would really appreciate to be able to disable sound after connection. Usually I wait for it and I see that terminal is connected. So the sound is redundant for me, it only disturbs my neighbourhood.

**Comment by tkedwards** [comment]
It's usually annoying, and unnecessary, to have that sound

**Comment by normanr** [comment]
Fwiw: my work around is to set ConnectBot notifications to silent.

**Comment by desaster** [comment]
Changing the notification importance to low seems to bring silence. I don't know if there's any drawbacks to this.

```java
diff --git a/app/src/main/java/org/connectbot/service/ConnectionNotifier.java b/app/src/main/java/org/connectbot/service/ConnectionNotifi
er.java
index ea33d42d..c7b7c1c8 100644
--- a/app/src/main/java/org/connectbot/service/ConnectionNotifier.java
+++ b/app/src/main/java/org/connectbot/service/ConnectionNotifier.java
@@ -79,7 +79,7 @@ public class ConnectionNotifier {
        @TargetApi(Build.VERSION_CODES.O)
        private void createNotificationChannel(Context context, String id) {
                NotificationChannel nc = new NotificationChannel(id, context.getString(R.string.app_name),
-                               NotificationManager.IMPORTANCE_DEFAULT);
+                               NotificationManager.IMPORTANCE_LOW);
                getNotificationManager(context).createNotificationChannel(nc);
        }
 
```

But looks like this can also be changed by user on per-app basis:

https://developer.android.com/guide/topics/ui/notifiers/notifications

**Comment by bmomjian** [comment]
Yeah, having it beep or vibrate on connection is clearly an odd behavior, and having to turn off notifications to fix that is also undesirable.  This behavior should be changed.
